actionmailer/lib/action_mailer/base.rb

```diff
@@ -434,6 +434,7 @@ def _protected_ivars # :nodoc:
     helper ActionMailer::MailHelper
 
     private_class_method :new #:nodoc:
+    private_class_method :find_class #:nodoc:
```

Comment:
I'd rather move this to right after the find_class definition. #new is here because it makes the existing Ruby method private, which means it's not an implementation of this class. Does it make sense?